### DISCUSSION TITLE
如何只引用文献title

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#154] [Q&A]
你好，有些文献，我只想引用文献的title，所以我使用了`\citetitle`，但是这个方式有两个问题：

1. 文中文献出现时，颜色、字体大小、斜体和全文不一致
2. 无法超链接到文档底部的参考文献

有办法解决这两个问题吗？
谢谢

**Comment by sikouhjw** [comment]
1. 提问请给 MWE
2. 颜色、字体大小没有不一致
3. 西文用出现书名是使用斜体作为标记，biblatex 提供的 `\citetitle` 在输出英文书名时，将其用斜体表示，输出中文书名时，用双引号 `“”` 表示
4. 你可以向 `texdoc biblatex` 发 [issue](https://github.com/plk/biblatex/issues)，提出需求添加超链接

**Reply by earthmoon** [reply]
你好，确实如你所说，只是

> @standard{RN110,
   author = {中华人民共和国卫生部 and 中国国家标准化管理委员会},
   title = {生活饮用水卫生标准：GB 5749 - 2006},
   publisher = {中国质检出版社},
   pages = {10},
   month = {07/01/2007},
   year = {2007},
   entrysubtype = {standard},
   url = {http://www.nhc.gov.cn/cmsresources/zwgkzt/wsbz/new/20070628143525.pdf}
> }

这个在使用\citetitle 引用时，被当作英文处理了。
因为我们都是使用武汉大学的模板，所以没有MWE，只需要在模板中输入几行代码就可。

**Reply by sikouhjw** [reply]
1. 如果被识别为英文，加上 `langid = {chinese},`
2. 任何问题都需要 MWE，即使只需要几行代码，也需要说明你是在什么条件下的结果

**Reply by earthmoon** [reply]
好的，谢谢！
